### ISSUE TITLE
Processors "jump" to another stream when using 2 splitters

### ISSUE THREAD

**Issue by biomath** [open-ephys/plugin-GUI#436]
Hi,

I'm running v0.5.4 (this issue was also present on 0.5.3) on Windows 10 with the Intan Rec. Controller.

When I save a configuration (attached) using 2 splitters (screenshot 1), after reloading the same configuration, processors on one side of the second splitter "jump" to the end of the first splitter  (screenshot 2). This happens regardless of what kind of processor I put on that end.

Let me know if you need any more information/files.

Thanks!

Screenshot 1:
![image](https://user-images.githubusercontent.com/7768262/115606576-de743180-a2b1-11eb-89bc-4fb0dc9ff758.png)

Screenshot 2:
![image](https://user-images.githubusercontent.com/7768262/115606691-05cafe80-a2b2-11eb-97e9-cb51cbecf898.png)

Config file:
[MML_NN_A4x16-Poly2-5mm-20s-lin-160_054.zip](https://github.com/open-ephys/plugin-GUI/files/6353252/MML_NN_A4x16-Poly2-5mm-20s-lin-160_054.zip)

**Comment by jsiegle** [comment]
Thanks, this is helpful! We'll get this fixed before the next release.

**Comment by jsiegle** [comment]
Figured out what was causing this...when selecting a processor with an upstream splitter, the splitters were not getting updated to point toward the correct branch of the signal chain. This was also making it so selecting a processor in the Graph View didn't necessarily make its editor visible.

After fixing this, the signal chain in your example re-loads fine (as well as some more complex signal chains I tested).
